frontend/src/views/Bridge.vue

```diff
@@ -74,9 +87,13 @@
             </b-button>
           </div>
           <div class="p-2">
-            <b-button :disabled="selectedNftId === '' ||
-            (currentTransferNFTId === selectedNftId && transferStatus !== transferStates.restored) ||
-            !storedNftsIds.includes(String(selectedNftId))"
+            <b-button variant="primary" @click="nftType = 'shield'; selectedNftId = ''; getStoredIds()"
```

Comment:
I would extract the @click to a method, as it has 3 actions going on here

Reply:
lol, I tagged some guy by accident
![image](https://user-images.githubusercontent.com/34208222/151954033-00952b7d-1b54-4de2-b1ef-a08bf68a3821.png)

If you're reading it, cheers mate :)